uber/badge_funcs.py

```diff
@@ -26,58 +26,61 @@ def get_badge_type(badge_num):
 
 
 def detect_duplicates():
-    subject = EVENT_NAME + ' Duplicates Report for ' + localized_now().strftime('%Y-%m-%d')
-    with Session() as session:
-        if session.no_email(subject):
-            grouped = defaultdict(list)
-            for a in session.query(Attendee).filter(Attendee.first_name != '').options(joinedload(Attendee.group)).order_by(Attendee.registered):
-                if not a.group or a.group.status != WAITLISTED:
-                    grouped[a.full_name, a.email.lower()].append(a)
```

Comment:
More checking config at runtime instead of import time.  (If it's not clear what I mean by that, let me know and I'll explain the reasoning.)